react-common/styles/semantic-ui-overrides.less

```diff
@@ -0,0 +1,448 @@
+
+/*
+--- Override Semantic UI ---
+
+This file overrides the default semantic ui theming with our own theme variables.
+
+*/
+
+// Mixin allows easy setting for background, foreground, hover, and inverted colors for a button
+.buttonVariant(@bg; @fg; @hoverBg; @hoverFg) {
```

Comment:
i think this file should be in theme and not react-common, since react-common doesn't depend on semantic ui.